kratos/includes/constitutive_law.h

```diff
@@ -82,6 +82,11 @@ class KRATOS_API(KRATOS_CORE) ConstitutiveLaw : public Flags
     typedef std::size_t SizeType;
     typedef Geometry<Node < 3 > > GeometryType;
 
+    typedef array_1d<double, 6> StrainVectorType;
+    typedef array_1d<double, 6> StressVectorType;
+    typedef BoundedMatrix<double, 6, 6> VoigtSizeMatrixType;     // Constitutive Matrix
+    typedef BoundedMatrix<double, 3, 3> DimensionSizeMatrixType; // Def. gradient tensor
```

Comment:
```suggestion
    typedef BoundedMatrix<double, 3, 3> DimensionSizeMatrix3DType; // Def. gradient tensor
```

Reply:
to be Honest i don't love this name.

what about "DeformationGradientType"